packages/devtools_app/lib/src/screens/vm_developer/object_inspector/vm_object_model.dart

```diff
@@ -2,6 +2,9 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file or at https://developers.google.com/open-source/licenses/bsd.
 
+/// @docImport '../../vm_developer/object_inspector/vm_simple_list_display.dart';
```

Comment:
Does the following not work?

```suggestion
/// @docImport 'vm_simple_list_display.dart';
```

Reply:
🤦  Fine I'll take your simpler path :P